src/Cerbere.vala

```diff
@@ -52,25 +53,23 @@ public class Cerbere.App : Application {
         hold ();
     }
 
-    private void register_session_client () {
+    private bool register_session_client () {
         if (sm_client != null)
-            return;
+            return false;
```

Comment:
Shouldn't this be returning true since sm_client already available and probably connected?

Reply:
The whole intention here was to return `false` to signal "I couldn't initialize `sm_client` as requested" and let `startup()` exit early. This all doesn't matter much since `startup()` _should_ be invoked exactly once, but having this branch return `false` would prevent repeated invocations from starting the monitored processes more than once.